src/common/helpers/repeatableFields.helper.ts

```diff
@@ -28,3 +29,30 @@ export const checkRepeatableGroup = ({
 
   return isRepeatableGroup;
 };
+
+export const checkRepeatableSubcomponent = ({
+  schema,
+  entry,
+  isDisabled,
+}: {
+  schema: Map<string, SchemaEntry>;
+  entry: SchemaEntry;
+  isDisabled: boolean;
+}) => {
+  const { type, path, constraints } = entry;
+  const isRepeatable =
+    !isDisabled &&
+    constraints?.repeatable &&
+    // show "Duplicate" button only for Provision Activity's subcomponents
+    // TODO: change or remove to display "Duplicate" button for other groups
+    findParentEntryByProperty({
+      schema,
+      path,
+      key: 'uriBFLite',
+      value: BFLITE_URIS.PROVISION_ACTIVITY,
+    }) &&
+    // TODO: remove this condition after updating the profile
```

Comment:
isn't the profile already was updated?

Reply:
May be, but we still have "repeatable" value as "true" for "Search place of publication (to/from 008)"